css/style.css

```diff
@@ -614,23 +622,30 @@ button.button-inline:hover {
 
 .colorselect {
 	overflow: hidden;
```

Comment:
no think that is a leftover from using floats before using flexbox with this PR.